doc/content/guides/connecting-gateways/tektelickonamicro/_index.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "Tektelic Kona Micro IoT LoRaWAN Gateway"
+description: ""
+weight: 1
+---
+
+Tektelic Kona Micro IoT LoRaWAN Gateway is an 8 Channel LoRaWAN gateway, whose technical specifications can be found in [the official documentation](https://www.thethingsnetwork.org/docs/gateways/tektelic/micro.html). This page guides you to connect it to {{% tts %}}.
```

Comment:
It feels a bit weird to link to call ttn.org official documentation for the Tektelic Kona Micro